OrbitGl/AccessibilityInterfaceMock.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TESTS_ACCESSIBILITY_INTERFACE_MOCK_H_
+#define ORBIT_GL_TESTS_ACCESSIBILITY_INTERFACE_MOCK_H_
+
+#include "OrbitGlAccessibility.h"
+
+namespace orbit_gl {
+
+class TestA11yImpl : public orbit_gl::GlAccessibleInterface {
```

Comment:
I've removed the "A11y" abbreviation everywhere now since I also was on the fence about this... Changed the name of the class to match the filename.

Regarding Fake vs Mock: I'm not sure if there's a suggested / accepted definition that we're following? If anything, isn't this a Stub? I've heard different definitions now, but "Fake" is either used as a super-set of mock & stub, or (e.g. https://martinfowler.com/articles/mocksArentStubs.html) for a class that actually behaves like the real thing. In this case, I'm exposing hard-coded information, which would make it a Stub... but I'm also verifying behavior (registration with the InterfaceRegistry)...

This sounds like I have a strong opinion, but I actually don't :-) Would be good to have a definition we use across our tests, though.

Reply:
Ok, following this definitions I agree... renamed the class and the header file.